reconcile/utils/oc.py

```diff
@@ -1322,7 +1325,10 @@ def get_items(self, kind, **kwargs):
             for resource_name in resource_names:
                 try:
                     item = obj_client.get(
-                        name=resource_name, namespace=namespace, label_selector=labels
+                        name=resource_name,
+                        namespace=namespace,
+                        label_selector=labels,
+                        _request_time=REQUEST_TIMEOUT,
```

Comment:
I think the name of the argument might be different? Looking at the following:

- [Client-side timeout (kwargs['_request_timeout'] = n)](https://github.com/kubernetes-client/python/blob/master/examples/watch/timeout-settings.md#client-side-timeout-kwargs_request_timeout--n)

```suggestion
                        _request_timeout=REQUEST_TIMEOUT,
```

Reply:
updated to `_request_timeout` according to https://github.com/kubernetes-client/python/pull/1732